packages/flutter_blue/proto/flutterblue.proto

```diff
@@ -0,0 +1,215 @@
+// Copyright 2017, Paul DeMarco.
+// All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+syntax = "proto3";
+
+option java_package = "com.pauldemarco.flutter_blue";
+option java_outer_classname = "Protos";
+option objc_class_prefix = "Protos";
+option optimize_for = LITE_RUNTIME;
+package proto.gen;
+// Wrapper message for `int32`.
```

Comment:
Apparently this file is not an exact copy of https://github.com/pauldemarco/flutter_blue/blob/master/protos/flutterblue.proto. Any reason?

Reply:
Generated file from the original did not contain any namespaces and it resulted in name collisions.

Reply:
Okay. Anyway please add a line break before line 12.